### PR TITLE
Set the current working dir for instruments

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Lets do it like Apple recommends in https://help.apple.com/instruments/mac/current/#/devb14ffaa5

Hopefully addresses https://github.com/appium/appium/issues/13048

### CODE REVIEW

**Comment by umutuzgur** [comment]
Could you follow the conventional commits strategy?

File: lib/commands/performance.js [hunk 1]
```diff
@@ -112,8 +117,8 @@ commands.mobileStartPerfRecord = async function mobileStartPerfRecord (opts = {}
       args.push('-p', pid);
     }
   }
-  const proc = new SubProcess('instruments', args);
-  log.info(`Starting 'instruments' with arguments: ${args.join(' ')}`);
+  const proc = new SubProcess(instrumentsPath, args);
+  log.info(`Starting ${instrumentsPath} with arguments: ${args.join(' ')}`);
```
Comment (imurchie) [inline lib/commands/performance.js]: This should still be in single quotes, to more easily see the value and any problems in the path.

Comment (mykola-mokhnach) [reply lib/commands/performance.js]: fixed

**Comment by mykola-mokhnach** [comment]
@umutuzgur I'll set the proper message upon squashing to master

**Pull request merged**